BasicVideoChat/README.md

```diff
@@ -0,0 +1,72 @@
+# BasicVideoChat
+
+This sample application shows how to connect to an OpenTok session,
+publish a stream, and subscribe to multiple streams for both iOS and
+Android using the OpenTok React Native API.
+
+## Setup
+
+1. Clone this repo
+
+```
+git clone https://github.com/opentok/opentok-react-native-samples
```

Comment:
@jraleman I don't think we should add the link to the repo because people might want to clone with SSH instead of HTTPS. We should remove the link.

Reply:
Yes, that sounds logical.